x-pack/plugins/maps/public/connected_components/mb_map/transform_request.ts

```diff
@@ -32,7 +50,7 @@ export function transformRequest(url: string, resourceType: string | undefined)
     };
   }
 
-  if (resourceType === 'Tile' && url.startsWith(GETTILE)) {
+  if (resourceType === 'Tile' && pathOrUrl.startsWith(GETTILE)) {
```

Comment:
We do not have to test the entire URL. If its easier, we could just test that URL contains path `MVT_GETTILE_API_PATH`